### ISSUE TITLE
send to multiple device in gideros

### ISSUE THREAD

**Issue by keszegh** [pkulchenko/ZeroBraneStudio#531]
in the latest gideros there is a feature to send to more devices at once (of course debugging info comes from only one of them). would this be possible with zbs?
e.g. macro:shell(ide.config.gideros = {remote = {"192.168.1.101","102.168.0.5"}}
or something similar.
gideros studio also finds automatically connected devices, so it's a bit even more automatic, but that's not that important.

**Comment by pkulchenko** [comment]
@keszegh, how does it work with Gideros? Does it connect to them one-by-one? Or does it send both addresses with one command? Do you know how `grbridge` command looks like for these addresses?

**Comment by keszegh** [comment]
sorry, i have no clue at all, either you can try to find it in the source code: https://github.com/gideros/gideros , or if you ask on the forum then developers will surely be able to answer all your questions. thanks for checking out.

**Comment by pkulchenko** [comment]
That's okay, I can check the source, but how would you use that feature? How would I test that it's working?

**Comment by keszegh** [comment]
if all goes well what one hopes to happen is something like:
-a gideros player on the developing pc starts with the app in it
-in a gideros player started on another device (e.g. an android phone) the app starts
usecase: i test a multiplayer (over local network) app so i need to constantly run on two devices to test my app; also others find it useful for checking immediately how the app looks on different screen sizes (e.g. the ide sends the app to run on 10 phones at the same time)

**Comment by pkulchenko** [comment]
@keszegh, are you sure it's the supported functionality? I don't see anything to this effect in [gdrbridge](https://github.com/gideros/gideros/blob/master/gdrbridge/main.cpp) source.

**Comment by keszegh** [comment]
probbly it's done from the gideros studio, there is no added functionality for gdrbridge, maybe it helps that these two commits take care of it:
https://github.com/gideros/gideros/commit/bd752bd01e95c52a53ad71ebbb583e3d2f411593
https://github.com/gideros/gideros/commit/bcb65a1c67f0aa2395525da81973195ddae69b98

**Comment by pkulchenko** [comment]
@keszegh, indeed; it looks like the first commit that added `MainWindow::startAllPlayers`. I'll check if it can be integrated... Patches/ideas welcome thought ;). If it's a matter of sending multiple `setip` commands to `gdrbridge`, it's not too difficult, but as far as I remember, next command resets the value of the previous one. Can you confirm that running `gdrbridge setip 192.168.1.101; gdrbridge setip 102.168.0.5; gdrbridge play projectfile` actually plays the project on two IP addresses? You may need to add `gdrbridge isconnected` before the `play` command to give it some time to connect...

**Comment by keszegh** [comment]
i tried and it did not seem to work but i don't even know how to make a while loop in a batch file, so i did not use gdrbridge isconnected. i tried by adding a gdrbridge play projectfile after the first setip too but that also does not work.
in any case i asked this on the forum, it's probably faster and easier then guessing:
http://giderosmobile.com/forum/discussion/6022/send-project-to-all-visible-players-using-gdrbridge

**Comment by pkulchenko** [comment]
@keszegh, thanks for posting this. I'll wait for someone to respond...
